Fixed obvious indentation error in InvalidResponseRecievedException c…

thozza opened the pull request:
Fixed obvious indentation error in InvalidResponseRecievedException class definition

Signed-off-by: Tomas Hozza <thozza@gmail.com>

dhoomakethu commented:
thanks for the PR @thozza , could you raise the PR against the riptideio:dev branch instead of master ?

thozza commented:
@dhoomakethu I rebased the change on top of dev branch and changed the base for PR. Everything should be all set. One additional question... Is there any place for discussion (like IRC channel or mailing list)?

dhoomakethu commented:
you can join https://gitter.im/pymodbus_dev/Lobby
